packages/stats/gbstats/gbstats.py

```diff
@@ -639,3 +640,14 @@ def process_experiment_results(data: Dict[str, Any]) -> list[Dict[str, Any]]:
                         )
                     )
     return results
+
+
+def process_multiple_experiment_results(
+    data: List[Dict[str, Any]]
+) -> list[Dict[str, Any]]:
+    results: List[Dict] = []
+    for exp_data in data:
+        exp_data_proc = ExperimentDataForStatsEngine(**exp_data)
```

Comment:
maybe wrap each iteration in a `try/except`?

Reply:
Done.